components/SetUrl.coffee

```diff
@@ -0,0 +1,18 @@
+noflo = require 'noflo'
+
+exports.getComponent = ->
+  c = new noflo.Component
+  c.inPorts.add 'in',
+    datatype: 'string'
+  c.outPorts.add 'out',
+    datatype: 'bang'
+
+  noflo.helpers.WirePattern c,
+    async: true
+    forwardGroups: false
+  , (data, groups, out, callback) ->
+    # This will in effect cause a NoFlo network stop as the app
```

Comment:
We're using this to redirect to GitHub, too. The URL query cleanup we do post-login can be done down the line with pushState